dali/python/nvidia/dali/ops/_signatures.py

```diff
@@ -356,7 +367,7 @@ def _gen_fn_signature(schema, schema_name, fn_name):
     pylance, resulting again in the Union[DataNode, List[DataNode]] return type (for single output),
     keeping us in the MIS realm in the simple case.
     """
-    return inspect_repr_fixups(f"""
+    return f"""
 @overload
```

Comment:
Why not have some `if` here and only append the second case when relevant? As-is there's some rather unnecessary repetition (the signature with no inputs is identical to the non-MIS one).

Reply:
It was more readable for me to spell out the:
```
@overload
<signature>

@overload
<signature>
```
rather than hide the `@overload` behind yet another condition, considering the fact, that the multiline strings are not the most readable thing on earth due to the significant indentations.